pkg/artifactory/resource_artifactory_remote_helm_repository.go

```diff
@@ -7,9 +7,10 @@ import (
 
 var helmRemoteSchema = mergeSchema(baseRemoteSchema, map[string]*schema.Schema{
 	"helm_charts_base_url": {
-		Type:         schema.TypeString,
-		Required:     true,
-		ValidateFunc: validation.IsURLWithHTTPorHTTPS,
+		Type:             schema.TypeString,
+		Optional:         true,
+		Default:          "",
+		ValidateDiagFunc: validation.ToDiagFunc(validation.Any(validation.StringIsEmpty, validation.IsURLWithHTTPorHTTPS)),
```

Comment:
So, it can be empty, or it can be http?

Reply:
Yes. 
On negative cases, the first validation is shown. So, Made IsURLWithHTTPorHTTPS as the first validation check followed by StringIsEmpty check.

Tested following scenarios:
Blank input to helm_charts_base_url argument
Without helm_charts_base_url argument
Valid helm_charts_base_url argument (http and https)
Invalid helm_charts_base_url argument (like http1)

Reply:
Please add these test cases to `resource_artifactory_remote_repository_test.go` 😄